CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 
 * `skbio.metadata.IntervalMetadata.drop` has a new boolean parameter `negate` to indicate whether to drop or keep the specified `Interval` objects.
 
+* `skbio.stats.distance` adding function permdisp. This fixes issue-1228
```

Comment:
Change last sentence to: 

This fixes [issue 1228](https://github.com/biocore/scikit-bio/issues/1228).